pipelines/release/docker/build_stack.groovy

```diff
@@ -76,7 +76,13 @@ notify.wrap {
     // should be removed after dropping support for dockerhub
   if (extraDockerTags) {
     // manual constructor is needed "because java"
+    // since we no longer need some of the current extraDockerTags,
```

Comment:
Would you consider removing the backward-compatibility-only `7-stack-lsst_distrib-*` from the nightly/weekly/official release pipelines and instead adding it back in after line 73 in this function using something like this:
```
def extraTagList = Arrays.asList(extraDockerTags.split())
registryTags += extraTagList
extraTagList.each { tag ->
    registryTags += "7-stack-lsst_distrib-${tag}"
}
```
Then the hard-coded removals below would not be needed, and the extra `each` loop could be removed after dropping docker.